requirements.txt

```diff
@@ -6,11 +6,11 @@ gitpython==0.3.2.1
 jinja2
 jsonschema>=2.3.0
 kombu
-mongoengine>=0.8.7,<0.9
+pymongo<3.0
+mongoengine>=0.9
```

Comment:
Yes. I need the "modify" function in 0.9.0 that supports atomic update in mongoengine document. There's an issue with pymongo >= 3.0 and will throw an exception "False is not a read preference".    Reference: https://github.com/MongoEngine/mongoengine/issues/935  pip installs mongoengine 0.9.0.  The fix seems to be in 0.9.1.